Documentation/ReleaseNotes/Production-Release-Notes-TEMPLATE.md

```diff
@@ -1,7 +1,8 @@
 <h1 align="center" style='color:mediumseagreen;font-weight:bold'>
-VersionMiner Release Notes - [add-prod-release-here]</h1>
+    VersionMiner Production Release Notes - [TODO: add-prod-release-here]
```

Comment:
Do you need this TODO here? 
```suggestion
    VersionMiner Production Release Notes - [TODO: add-prod-release-here]
```

Reply:
The TODO is supposed to be in the release notes template.